Source/Build Specs/MeasurementLink Generator.vipb

```diff
@@ -1,7 +1,7 @@
-<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-10-23 20:38:22" Creator="lvadmin" Comments="" ID="25af46a7b45831f84100f328a8769c33">
+<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-11-30 14:28:49" Creator="lvadmin" Comments="" ID="3fca94ae292362bfc4a0ea9a8c105533">
   <Library_General_Settings>
     <Package_File_Name>NI_MeasurementLink_Generator</Package_File_Name>
-    <Library_Version>1.3.0.2</Library_Version>
+    <Library_Version>1.3.0.4</Library_Version>
```

Comment:
Shouldn't this be .3? Same with the MeasurementLink Service. We have only released a .1. So this current release you're working on will be .2. Which means the next release (reflected in this PR) should be .3.